package.json

```diff
@@ -7,7 +7,8 @@
     "build-assets": "make js && make css",
     "check-bundles": "npm run build-assets && bundlesize",
     "lint:fix": "stylelint --syntax less --fix static/css/",
-    "test": "npm run check-bundles && stylelint  --syntax less static/css/"
+    "test": "npm run check-bundles && stylelint  --syntax less static/css/",
```

Comment:
test job should also run lint:js - this will mean its run in CI!

it probably makes sense to pull out 
`stylelint  --syntax less static/css/`
into 
"npm run lint:css" for consistency!